### PR TITLE
Registration updates

### LINKED ISSUE #15: Add timeout to Pretix API calls
Adding time out when connecting to Pretix

May relate to #3 - I think it can be close together when we start working on Pretix integration, or basic integration is added first and sort this out later.

### LINKED ISSUE #17: Improve registration form UX
I am not too sure about this one, maybe @Kislovskiy can elaborate, or maybe it means just a simple check on if the input is valid 

(e.g. no emojis or not allowing characters in names - what is valid depends on what Pretix said is valid; 
checking if the ticket identification number has enough characters etc.)

### PR DESCRIPTION
**Pull request by NMertsch**
- Close #17: Require order ID of exactly 5 characters, convert to uppercase for ticket lookups
- Close #15: Add timeout of 5 seconds for Pretix API requests
- Registration form: Improve label and placeholder to be more explicit (see screenshots)
- On bot shutdown: Replace registration form with message "currently offline" (see screenshots)

Bot online (unchanged):
image

Registration form
image

Bot offline: 
image

### CODE REVIEW

File: EuroPythonBot/registration/cog.py [hunk 1]
```diff
@@ -145,13 +145,16 @@ def __init__(self, bot: Client):
 
     @commands.Cog.listener()
     async def on_ready(self):
+        print("HERE!!")
```
Comment (Kislovskiy) [inline EuroPythonBot/registration/cog.py:148]: Some debug leftover?

File: EuroPythonBot/registration/cog.py [hunk 2]
```diff
@@ -179,9 +176,19 @@ async def cog_load(self) -> None:
...
+        _logger.info("Canceling periodic pretix update task.")
         self.fetch_pretix_updates.cancel()
 
+        _logger.info("Replacing registration form with 'currently offline' message")
+        reg_channel = self.bot.get_channel(config.REG_CHANNEL_ID)
+        await reg_channel.purge()
+        await reg_channel.send(
+            embed=create_welcome_message(
+                "The registration bot is currently offline. "
+                "We are sorry for the inconvenience and are working hard on fixing the issue."
```
Comment (Kislovskiy) [inline EuroPythonBot/registration/cog.py:188]: Maybe?
```suggestion
                "We apologize for the inconvenience and are working hard to fix the issue."
```

File: EuroPythonBot/registration/cog.py [hunk 3]
```diff
@@ -33,20 +33,20 @@ def __init__(self, parent_cog: RegistrationCog):
...
+        min_length=5,
+        max_length=5,
+        placeholder="e.g. 'ABC12'",
     )
 
     name_field = discord.ui.TextInput(
-        label="Full Name",
+        label="Full Name (As on your ticket)",
         required=True,
         min_length=3,
```
Comment (Kislovskiy) [inline EuroPythonBot/registration/cog.py:46]: I think we could set `min_length` to 1; I have a friend with the family name `U`.

**Comment by NMertsch** [comment]
Closing to split it into smaller PRs, one per issue.

**Pull request closed without merging**